command_handler.py

```diff
@@ -28,30 +28,31 @@ def validate_paths(dpct_project_path, destination_path):
     generates exceptions if directory of destination_path is not empty
     or if dpct_project_path directory contains no (nested) .dp.cpp or .dp.h files
     '''
-    if os.path.exists(dpct_project_path) is False:
-        r = "The dpct project path does not exist."
-        return r
-    filenames = os.listdir(dpct_project_path)
+    try:
+        if os.path.exists(dpct_project_path) is False:
+            raise Exception(dpct_project_path + " does not exist.")
+    except Exception as e:
+        return e
```

Comment:
```suggestion
    if os.path.exists(dpct_project_path) is False:
        raise Exception(dpct_project_path + " does not exist.")
```
No need to do try except if you then raise the exception anyways